react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -43,6 +43,7 @@ export interface DeckGLMapProps {
      */
     checkDatafileSchema?: boolean;
     onMouseEvent?: (event: MapMouseEvent) => void;
+    colorMapping?: (t: number) => string | string[];
```

Comment:
This should be a layer prop. We need to be able to set separate color mapping to each layer.